### ISSUE TITLE
Adding translations works but shows "Unknown error"

### ISSUE THREAD

**Issue by I-Valchev** [bolt/core#710]
Adding a kay-value translation on `/bolt/_trans` creates the record, but shows "Unknown error" message.

![Screenshot 2019-11-06 at 13 20 35](https://user-images.githubusercontent.com/7093518/68306051-1630ee80-00a9-11ea-9930-182e1f771a07.png)

**Comment by UtechtDustin** [comment]
This site is created by the [symfony php-translation bundle](https://github.com/php-translation/symfony-bundle) and the error `Unknown error` is caused by the missing api key from loco.

So i guess it has to be fixed in the bundle repository.

**Comment by bobdenotter** [comment]
@UtechtDustin Good catch.. In my local checkout, i have the "official" key that connects it to https://localise.biz/ .. Ivo, I'll paste you the key, and I assume then it'll work for you.. 

However, we should not put this key in our repo. Ideally, it would show a proper warning for those who don't have it.. 🤔

**Comment by macintoshplus** [comment]
Hi @I-Valchev, have you some information about bolt translation project on https://localise.biz/ ?

Please, contact us on bolt slack @jean-baptiste.
